consensus/spos/debug/equivalentMessagesDebugger.go

```diff
@@ -49,6 +52,9 @@ func dataToString(data map[string]*consensus.EquivalentMessageInfo) string {
 		line := []string{
 			hash,
 			fmt.Sprintf("%d", info.NumMessages),
+			fmt.Sprintf("%T", info.Validated),
```

Comment:
`%T` ? This will print the type of info.Validated

Reply:
typo, updated to `%t`